python/lsst/obs/base/gen2to3/repoWalker/scanner.py

```diff
@@ -167,15 +168,18 @@ class DirectoryScanner:
     ----------
     log : `Log`, optional
         Log to use to report warnings and debug information.
+    progress : `Progress`, optional
+        Object to use to report incremental progress.
     """
-    def __init__(self, log: Optional[Log] = None):
+    def __init__(self, log: Optional[Log] = None, progress: Optional[Progress] = None):
         self._files = []
         self._subdirectories = []
         if log is None:
             log = Log.getLogger("obs.base.gen2to3.walker")
         self.log = log
+        self.progress = None
```

Comment:
Why is this `None` and not using the parameter?

Reply:
GitHub's reaction dropdown list needs :facepalm:: because I've fixed this at least twice in fixup commits and keep blowing it away as I juggle branches. Glad you caught this one.